### ISSUE TITLE
Pull in Expected Streaming Service from TMDB to Prevent Grabbing Movies that are Streamable

### ISSUE THREAD

**Issue by slevitt5** [Radarr/Radarr#7659]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

I use Radarr to grab movies that are not available on my preferred streaming services (e.g., Netflix, Hulu, Amazon Prime Video, etc.) Radarr pulls all the movies that I add to my Trakt watchlist and adds them as Monitored. This is frustrating at times because then I have to manually check TMDB/JustWatch to see if the movie is available on a streaming service, then manually UNmonitor it if it is. And often times, I don't do this fast enough, so the movie has already been grabbed (even though I can watch it on a streaming service that I pay for).

### Describe the solution you'd like

Since Radarr is already integrated with TMDB, I think there is a super easy solution to this problem. Let's take this upcoming movie, Hellraiser, as an example. On the TMDB "Release Dates" page for this movie (https://www.themoviedb.org/movie/338947-hellraiser/releases?language=en-US), it shows that the expected Digital release date is 10/07/2022 (in the U.S.). And in the "Note" field, it shows that it will be released on Hulu. Here is a screenshot:

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/115101080/194112675-3d66389b-8688-4596-9325-4a48a8a270b7.png">

I would love if Radarr could pull this "Note" field in. Radarr already pulls in the "Digital Release Date", so I can't imagine it would be too challenging to also pull in the "Note."

At a minimum, I would love to see the "Note" below the movie poster. So, for example, it would show up under the Digital Release date in this screenshot:

<img width="161" alt="image" src="https://user-images.githubusercontent.com/115101080/194114388-dbc04881-f59c-44b6-95d7-1fbc12c58d40.png">

But what would be even better is if I could add criteria to my List Imports based on this "Note" field. In other words, only "Enable Automatic Add" if Digital Release "Note" field does NOT contain "Netflix" or "Hulu" or etc.

### Describe alternatives you've considered

I've done quite a bit of digging online, and I stumbled across Excludarr (https://github.com/haijeploeg/excludarr). This solution appears to be EXACTLY what I'm looking for. The issue is that I have no formal coding experience, and I cannot use it without a UI.

### Anything else?

In summary, I just really want to prevent Radarr from grabbing movies that I can watch on the streaming services I already pay for.

**Comment by Qstick** [comment]
Dupe #3333
